### PR TITLE
Added notification creation functionality

### PR DESCRIPTION
**Pull request by Edward-Boguslavsky**
Whenever an application is submitted, the user now gets a notification with some info about their application. Also, the notifications list is now sorted from latest to earliest

### CODE REVIEW

File: launchpad-frontend/src/jobsPage/jobsPage.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ import { ApplyButton } from "../components/applicationPopUp";
 import { useParams, useNavigate } from "react-router-dom";
 import CircularProgress from "@mui/material/CircularProgress";
 
+// Current logged in user's ID
+let currentUserId = -1;
```
Comment (nikkikapadia) [inline launchpad-frontend/src/jobsPage/jobsPage.js:26]: you don't need this

File: launchpad-frontend/src/jobsPage/jobsPage.js [hunk 2]
```diff
@@ -457,8 +463,10 @@ function JobExpanded({ postingSelected, updateList, postings, saved }) {
                   }}
                 >
                   <ApplyButton
+                    userID={currentUserId}
```
[the remark this answers is not in the document]
Comment (Edward-Boguslavsky) [reply launchpad-frontend/src/jobsPage/jobsPage.js]: Fixed!

**Review by nikkikapadia — APPROVED** [review]

File: launchpad-frontend/src/components/applicationPopUp.js [hunk 3]
```diff
@@ -71,6 +71,22 @@ export function ApplyButton({ postingID, companyName }) {
           setSubmissionStatus('success');
           setApplicationId(responseData.applicationId);
           console.log('Submission successful. Application ID:', responseData.applicationId);
+
+          // Create subject and body for submission notification
+          const subject = `Application Received - ${companyName}`;
+          const body = `Thank you for applying to the \"${positionName}\" position at ${companyName}. Please check your notifications for more updates; you should hear back from them soon!<br>Best of luck,<br>LaunchPad`;
```
[the remark this answers is not in the document]
Comment (zanq-mw) [reply launchpad-frontend/src/components/applicationPopUp.js:77]: if u rtaking them out take out the quotes too tho lmao

[the remark this answers is not in the document]
Comment (Edward-Boguslavsky) [reply launchpad-frontend/src/components/applicationPopUp.js:77]: I added the \" there to help separate the long name from the rest of the text. The quotes will appear in the notification
image

**Pull request merged**